services/minespace-web/src/components/dashboard/mine/inspections/InspectionsTable.js

```diff
@@ -1,39 +1,124 @@
-import React from "react";
-import { Table } from "antd";
+import React, { useState } from "react";
+import { Table, Button } from "antd";
+import moment from "moment";
+import { formatDate, compareCodes } from "@common/utils/helpers";
 import PropTypes from "prop-types";
+import CustomPropTypes from "@/customPropTypes";
+import { RED_CLOCK } from "@/constants/assets";
 
 const propTypes = {
+  orders: CustomPropTypes.complianceOrders,
   isLoaded: PropTypes.bool.isRequired,
 };
 
-// TODO: Ensure column order is correct and if any columns should be added or removed.
+const defaultProps = {
+  orders: [],
+};
+
 const columns = [
-  { title: "Order No.", dataIndex: "order_no", key: "order_no", sorter: true },
-  { title: "Violation", dataIndex: "violation", key: "violation", sorter: true },
-  { title: "Report No.", dataIndex: "report_no", key: "report_no", sorter: true },
-  { title: "Inspection Type", dataIndex: "inspection_type", key: "inspection_type", sorter: true },
-  { title: "Inspector", dataIndex: "inspector", key: "inspector", sorter: true },
-  { title: "Order Status", dataIndex: "order_status", key: "order_status", sorter: true },
-
-  // NOTE: Brian: Can we include a tip that says "This is the date by which the violation will be rectified."?
-  { title: "Due", dataIndex: "due", key: "due", sorter: true },
+  {
+    title: "",
+    dataIndex: "overdue",
+    render: (text, record) => (
+      <div title="Overdue">
+        {record.overdue && record.due_date !== null ? (
+          <img className="padding-small" src={RED_CLOCK} alt="Overdue Report" />
+        ) : (
+          ""
+        )}
+      </div>
+    ),
+  },
+  {
+    title: "Order No.",
+    dataIndex: "order_no",
+    key: "order_no",
+    render: (text, record) => <div title="Order #">{record.order_no || "-"}</div>,
+    sorter: (a, b) => (a.order_no > b.order_no ? -1 : 1),
```

Comment:
There is a const for '-', I believe its 'EMPTY_FIELD'

Reply:
We do not, EMPTY_FIELD is "N/A"